packages/api-fetch/README.md

```diff
@@ -85,4 +85,27 @@ const rootURL = "http://my-wordpress-site/wp-json/";
 apiFetch.use( apiFetch.createRootURLMiddleware( rootURL ) );
 ```
 
+### Custom fetch handler
+
+The `api-fetch` package uses `window.fetch` for making the requests but you can use a custom fetch handler by using the `setFetchHandler` method. The custom fetch handler will receive the `options` passed to the `apiFetch` calls.
+
+**Example**
+
+The example below uses a custom fetch handler for making all the requests with [`axios`](https://github.com/axios/axios).
+
+```js
+import apiFetch from '@wordpress/api-fetch';
+import axios from 'axios';
+
+apiFetch.setFetchHandler( ( options ) => {
+	const { url, path, data, method } = options;
+
+	return axios({
```

Comment:
Nitpick:

>Arrays and Function Calls
>Always include extra spaces around elements and arguments:

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#arrays-and-function-calls